reflect.go

```diff
@@ -91,11 +90,9 @@ func getDBFields(t reflect.Type) fieldMap {
 func (m fieldMap) getTraversals(names []string) [][]int {
 	var traversals [][]int
 	for _, name := range names {
-		f, ok := m[name]
-		if !ok {
-			panic(fmt.Errorf("db field '%s' has no mapping", name))
+		if f, ok := m[name]; ok {
+			traversals = append(traversals, f.Index)
```

Comment:
This new behavior deserves a test.

You should probably append an empty `[]int` slice if the name couldn't be found. It's probably also worthwhile to return some sort of error so that higher levels can be aware that `getTraversals` failed.